pkg/proxyagent/agent/agent.go

```diff
@@ -156,6 +156,8 @@ func GetClusterProxyValueFunc(runtimeClient client.Client, nativeClient kubernet
 			annotations[common.AnnotationKeyConfigurationGeneration] = strconv.Itoa(int(proxyConfig.Generation))
 		}
 
+		serviceEntryPointPort := proxyConfig.Spec.ProxyServer.Entrypoint.Port
```

Comment:
also for data-plane compatibility, reset it to 443 if it's zero

Reply:
Sure. Fixed.